crates/red_knot_python_semantic/resources/mdtest/import/case_sensitive.md

```diff
@@ -0,0 +1,126 @@
+# Case Sensitive Imports
+
+TODO: This test should use the real file system instead of the memory file system.
+
+Python's import system is case-sensitive even on case-insensitive file system. This means, importing
+a module `a` should fail if the file in the search paths is named `A.py`. See
+[PEP 235](https://peps.python.org/pep-0235/).
+
+## Correct casing
+
+Importing a module where the name matches the file name's casing should succeed.
+
+`a.py`:
+
+```py
+class A:
```

Comment:
nit: I think using a class named `A` in these tests is confusing for no benefit, when it could be named `C` or anything else